spec/src/main/asciidoc/microprofile-lra-spec.adoc

```diff
@@ -684,6 +684,10 @@ or may in the process of compensation (completion) the exception in these method
 result into failure condition (in JAX-RS this condition is represented by 
 500 return HTTP status code) which individual interpretation is left further unspecified.
 
+The non-JAX-RS participant methods MUST be invoked by the implementation of this specification 
```

Comment:
I would say

> ... invoked by the implementation of the specification **only** and the implementation MUST ...

otherwise the sentence seems a bit odd to me.